photon-api/src/main/scala/com/linkedin/photon/ml/optimization/RegularizationContext.scala

```diff
@@ -44,12 +44,10 @@ class RegularizationContext(val regularizationType: RegularizationType, val elas
     require((regularizationType == RegularizationType.ELASTIC_NET) || elasticNetParam.isEmpty,
       "Elastic net parameter can be specified only for elastic net regularization")
 
-    require(regularizationType != RegularizationType.ELASTIC_NET
-      || (elasticNetParam.isDefined && elasticNetParam.exists(p => 0.0d < p && p <= 1.0d))
-      || elasticNetParam.isEmpty,
-      s"""Elastic net regularization is specified, but elastic net param ($elasticNetParam.get)
-         | should be in interval (0,1].
-       """.stripMargin)
+    require(
+      regularizationType != RegularizationType.ELASTIC_NET || elasticNetParam.exists(p => 0.0d < p && p <= 1.0d),
+      s"Elastic net regularization is specified, but elastic net alpha " +
+          s"(${elasticNetParam.map(_.toString).getOrElse("NONE")}) is not in interval (0,1).")
```

Comment:
Information on bounds is lost in new comment. Previous comment specified that 0 was not allowed for p, but that 1 is allowed. Also, what happened to isEmpty?